docs/3.x/tech/release-notes/index.adoc

```diff
@@ -12,14 +12,57 @@ Hide Stuff here
 
 // REMEMBER TO ADD MIGRATIONS TO upgrades-patches
 
+[role=release-note]
+== Version 3.31.3
+_Released Tuesday November 9th, 2021_
+
+=== Fixed
+* When adding an automated user action for a user action that is not time based you are unable to save.
+** Resolves https://github.com/CleanSpeak/cleanspeak-issues/issues/14[Github Issue #14]
+* Restoring from backup when the array `WhitelistedURLs` has an entry that is a `String` instead of an `Object` throws an exception.
+** Resolves https://github.com/CleanSpeak/cleanspeak-issues/issues/18[Github Issue #18]
+* Editing a blacklist phrase and approving the change does not show in the audit log.
+** Resolves https://github.com/CleanSpeak/cleanspeak-issues/issues/19[Github Issue #19]
+* If an event has never been configured (enabled or disabled) for a webhook an exception may occur when that event occurs, and an attempt to send that event to the webhook occurs.
+** Resolves https://github.com/CleanSpeak/cleanspeak-issues/issues/29[Github Issue #29]
+* Attempting to action a user from the dashboard, and many other places is blocked by the fact that selecting an action does not update the rest of the UI. Specifically, the related controls are not properly hidden or shown, and the submit button is never enabled.
+** Resolves https://github.com/CleanSpeak/cleanspeak-issues/issues/31[Github Issue #31]
+* When submitting an escalation comment that is longer than 255 characters saving the escalation will fail.
+** Resolves https://github.com/CleanSpeak/cleanspeak-issues/issues/32[Github Issue #32]
+* When adding a moderation action with any one of location, sender display name, or receiver display name that is longer than 255 characters it will block them from being asynchronously added to the database. The action gets stuck in the queue and is never cleared or saved.
+** Resolves https://github.com/CleanSpeak/cleanspeak-issues/issues/33[Github Issue #33]
+* The OIDC integration generates invalid OAuth redirect URLs when starting the OAuth login process. The URLs have two `?` symbols, which make them invalid.
+** Resolves https://github.com/CleanSpeak/cleanspeak-issues/issues/36[Github Issue #36]
+* The css text color for `h3` inside certain tooltips is not correct. It's black on black, so you can't read the text.
+** Resolves https://github.com/CleanSpeak/cleanspeak-issues/issues/38[Github Issue #38]
+
+=== Changes
```

Comment:
We should use `Changes` instead of `Changed`, and these should go before fixes. 

However, I want to say this is a bug we fixed instead of a change. The default should never have been 0 because it means everything was a low, med, or high.

Reply:
Moving that entry emptied `Changes` so I removed that section.